Squest/authentication.py

```diff
@@ -0,0 +1,48 @@
+from django.core.exceptions import ImproperlyConfigured
+from django_auth_ldap.backend import LDAPBackend as LDAPBackend_
+
+
+class LDAPBackend:
+
+    def __new__(cls, *args, **kwargs):
+        try:
+            from django_auth_ldap.backend import LDAPSettings
+            import ldap
+        except ModuleNotFoundError as e:
+            if getattr(e, 'name') == 'django_auth_ldap':
+                raise ImproperlyConfigured(
+                    "LDAP authentication has been configured, but django-auth-ldap is not installed."
+                )
+            raise e
+
+        try:
+            from Squest import ldap_config
+        except ModuleNotFoundError as e:
+            if getattr(e, 'name') == 'ldap_config':
+                raise ImproperlyConfigured(
+                    "LDAP configuration file not found: Check that ldap_config.py has been created alongside "
+                    "configuration.py."
+                )
+            raise e
+
+        try:
+            getattr(ldap_config, 'AUTH_LDAP_SERVER_URI')
+        except AttributeError:
+            raise ImproperlyConfigured(
+                "Required parameter AUTH_LDAP_SERVER_URI is missing from ldap_config.py."
+            )
+
+        obj = LDAPBackend_()
+
+        # Read LDAP configuration parameters from ldap_config.py instead of settings.py
```

Comment:
the ldap_config.py file is loaded from [settings](https://github.com/HewlettPackard/squest/blob/master/Squest/settings/development.py#L126).
We should load only settings which come with ldap if activated